packages/jest-each/src/table/interpolation.ts

```diff
@@ -77,7 +77,7 @@ export function getPath(
   template: Template,
   [head, ...tail]: Array<string>,
 ): unknown {
-  if (!head || !template.hasOwnProperty || !template.hasOwnProperty(head))
+  if (!head || !template.hasOwnProperty || !Object.prototype.hasOwnProperty.call(template, head))
```

Comment:
@SimenB i am kinda confused with this one, this is not running i guess, could you just help me with this?

Reply:
drop the `!template.hasOwnProperty` call (and run prettier, which I assume is not needed after doing so)